### ISSUE TITLE
How to define types of statics with mongoose 5.11 ?

### ISSUE THREAD

**Issue by danez** [Automattic/mongoose#9678]
**Do you want to request a *feature* or report a *bug*?**

bug

**What is the current behavior?**

I cannot get type definitions for statics on Models in.

```
error TS2339: Property 'findOrCreateProject' does not exist on type 'Model<ProjectDoc>'.

93     const project = await Project.findOrCreateProject(projectName);

error TS2558: Expected 1 type arguments, but got 2.

42 export const Project = mongoose.model<ProjectDoc, ProjectModelDoc>("Project", ProjectSchema);
                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**If the current behavior is a bug, please provide the steps to reproduce.**

This is what I did previously (with 5.10 and @types/mongoose), but it does not work anymore. Model does not have two type arguments anymore:

```js
import mongoose, { Schema, Document, Model } from "mongoose";

export interface ProjectDoc extends Document {
  name: string;
}

interface ProjectModelDoc extends Model<ProjectDoc> {
  findOrCreateProject(name: string): Promise<ProjectDoc>;
}

const ProjectSchema = new Schema(
  {
    n: { type: String, required: true, unique: true, alias: "name" },
  },
  { _id: true, timestamps: { createdAt: "ca", updatedAt: "ua" } }
);

ProjectSchema.statics.findOrCreateProject = async function (
  name: string
): Promise<ProjectDoc> {
  ...
};

export const Project = mongoose.model<ProjectDoc, ProjectModelDoc>("Project", ProjectSchema);
```

**What is the expected behavior?**

No error

**What are the versions of Node.js, Mongoose and MongoDB you are using? Note that "latest" is not a version.**

mongoose: 5.11.5

**Comment by Jesusz0r** [comment]
I'm having the same problem. Do you have any solution for this?

**Comment by vkarpov15** [comment]
@Jesusz0r the fix hasn't been released yet, we're shipping v5.11.5 shortly.
